CHANGELOG.md

```diff
@@ -6,14 +6,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [v0.6.0] - 2021-06-24
+### Added
+- Threaded Async Executor for smol. ([#254](https://github.com/paritytech/substrate-archive/commit/36d955d379b1fdfb0ff063dce394d8a4d6430323))
```

Comment:
need to add #301 here and state that `async-std` has replaced `smol` ^

Reply:
Yeah, fixed this. Don't think it's actually breaking because the user never see's the executor when using the engine